Library/Formula/ploticus242.rb

```diff
@@ -0,0 +1,18 @@
+class Ploticus242 < Formula
+  desc "ploticus data display package"
+  homepage "http://ploticus.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/ploticus/ploticus/2.42/ploticus242_src.tar.gz?r=http%3A%2F%2Fsourceforge.net%2Fprojects%2Fploticus%2F%3Fsource%3Dtyp_redirect&ts=1449655936"
+  version "2.42"
+  sha256 "3f29e4b9f405203a93efec900e5816d9e1b4381821881e241c08cab7dd66e0b0"
+
+  def install
+    cd "src" do
+      system "make"
+      bin.install "pl"
+    end
+  end
+
+  test do
+    system "pl 2>&1 >/dev/null | grep 'usage: pl scriptfile'"
```

Comment:
The issue is that pl returns 1 so this way the test is not passed.

/Magnus

On Wed, Dec 9, 2015 at 11:43 AM, Baptiste Fontaine <notifications@github.com
> wrote:

> In Library/Formula/ploticus242.rb
> <https://github.com/Homebrew/homebrew/pull/46831#discussion_r47075121>:
>
> > +class Ploticus242 < Formula
> > +  desc "ploticus data display package"
> > +  homepage "http://ploticus.sourceforge.net/"
> > +  url "https://downloads.sourceforge.net/project/ploticus/ploticus/2.42/ploticus242_src.tar.gz?r=http%3A%2F%2Fsourceforge.net%2Fprojects%2Fploticus%2F%3Fsource%3Dtyp_redirect&ts=1449655936"
> > +  version "2.42"
> > +  sha256 "3f29e4b9f405203a93efec900e5816d9e1b4381821881e241c08cab7dd66e0b0"
> > +
> > +  def install
> > +    cd "src" do
> > +      system "make"
> > +      bin.install "pl"
> > +    end
> > +  end
> > +
> > +  test do
> > +    system "pl 2>&1 >/dev/null | grep 'usage: pl scriptfile'"
>
> Please use assert_match "usage: pl scriptfile", shell_output("#{bin}/pl
> 1>&1").
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/46831/files#r47075121>.
>